examples/forms.amp.html

```diff
@@ -206,15 +206,16 @@ <h4>Search (GET non-xhr cross origin - current host > httpbin.org)</h4>
 </form>
 
 <h4>Search (GET xhr same origin)</h4>
-<form method="get" action="/form/search-html/get" action-xhr="/form/search-json/get" target="_blank">
+<form method="get" id="xhr-get" action="/form/search-html/get" action-xhr="/form/search-json/get" target="_blank">
     <fieldset>
         <label>
             <span>Search for</span>
             <input type="search" name="term" required>
         </label>
         <input type="submit" value="Search">
-    </fieldset>
 
+    </fieldset>
+    <a on="tap:xhr-get.submit">link</a>
```

Comment:
nit: `<a>` tags that do not link to anywhere should be buttons, for accessibility and semantic correctness.